packages/block-library/src/post-tags/index.php

```diff
@@ -33,7 +34,7 @@ function render_block_core_post_tags( $attributes, $content, $block ) {
 		}
 
 		$output  = trim( $output, ' | ' );
-		$output .= '</div>';
+		$output .= '</div>'.( ! empty( $byline ) ? '<p class="wp-block-post-author__byline">' . $byline . '</p>' : '' );
```

Comment:
`byline` output should be inside the block and before the list output, as is in the editor. Also the class name should be about `post-tags`.